src/ptf/dataplane.py

```diff
@@ -49,23 +49,38 @@
 #except:
 #    pass
 
+dump_pkts_on_mismatch = False
```

Comment:
please do not use a global variable for this, you can use the `config` directory and update the command line parser